src/neo-vm/OpCode.cs

```diff
@@ -741,11 +731,43 @@ public enum OpCode : byte
         /// A map is taken from top of the main stack. The values of this map are put on top of the main stack.
         /// </summary>
         VALUES = 0xCD,
+        /// <summary>
+        /// An input index n (or key) and an array (or map) are taken from main stack. Element array[n] (or map[n]) is put on top of the main stack.
+        /// </summary>
+        PICKITEM = 0xCE,
+        /// <summary>
+        /// The item on top of main stack is removed and appended to the second item on top of the main stack.
+        /// </summary>
+        APPEND = 0xCF,
+        /// <summary>
+        /// A value v, index n (or key) and an array (or map) are taken from main stack. Attribution array[n]=v (or map[n]=v) is performed.
+        /// </summary>
+        SETITEM = 0xD0,
+        /// <summary>
+        /// An array is removed from the top of the main stack and its elements are reversed.
+        /// </summary>
+        REVERSEITEMS = 0xD1,
+        /// <summary>
+        /// An input index n (or key) and an array (or map) are removed from the top of the main stack. Element array[n] (or map[n]) is removed.
+        /// </summary>
+        REMOVE = 0xD2,
+        /// <summary>
+        /// Remove all the items from the compound-type.
+        /// </summary>
+        CLEARITEMS = 0xD3,
+
+        #endregion
+
+        #region Types
 
         /// <summary>
         /// Returns true if the input is null. Returns false otherwise.
         /// </summary>
-        ISNULL = 0xD0,
+        ISNULL = 0xD8,
+        [OperandSize(Size = 1)]
```

Comment:
Perhaps add to other comments here, one for `ISTYPE` and one for `CONVERT`.

In summary, all changes looks good to me, readability and safety improved.